src/app/backend/overview/overview.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2017 The Kubernetes Dashboard Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package overview
+
+import (
+	metricapi "github.com/kubernetes/dashboard/src/app/backend/integration/metric/api"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/common"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/config"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/dataselect"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/discovery"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/workload"
+	"github.com/kubernetes/dashboard/src/app/backend/search"
+	"k8s.io/client-go/kubernetes"
+)
+
+// OverviewObjectList is a list of objects present in a given namespace
+type OverviewObjectList struct {
+	search.AllObjectsInNamespace
+}
+
+func GetOverview(client *kubernetes.Clientset, metricClient metricapi.MetricClient,
```

Comment:
Can you add comments to all public methods/variables? They should start with method/variable name.